netsuite/10.0/modules/ROOT/pages/netsuite-connector-upgrade-migrate.adoc

```diff
@@ -10,96 +10,114 @@ Upgrade NetSuite Connector to version 10.0 to use the new features.
-| Update Record a| attributes a| Is now a *content* parameter
-| Upsert Record a| attributes a| Is now a *content* parameter
+|Operation | Parameter
+
+| Change Email a| newEmail
+| Change Password a| newPasword
+| Get Data Center URLs a| accountId
+| Get Async Result a| jobId
+| Get Custom Record a| internalId 
+|===
+
+The following operations are now *primary content* and *content* parameters:
+
+[%header%autowidth.spread]
+|===
+|Operation | Parameter 
+| Async Search a| content 
+| Async Add List a| recordAttributes 
+| Async Update List a| recordAttributes 
+| Async Upsert List a| recordAttributes
+| Update Invitee Status List a| inviteeStatuses 
+| Get Posting Transaction Summary 
+| Get Posting Transaction Summary 
+| Add File a| attributes 
+| Add Record a| attributes  
+| Get Deleted Records a| deletedDate 
+| Update Record a| attributes 
+| Upsert Record a| attributes 
 |===
 
 [[changed_sources]]
 == Changed Sources
 
-The three message sources were renamed.
+The three message sources were renamed:
 
 [%header%autowidth.spread]
 |===
 |Source name |Old XML name | New XML name
 
-| On New Object | `<netsuite:new-object-trigger ...` | `<netsuite:new-object-listener ...`
-| On Modified Object |  `<netsuite:modified-trigger ...` | `<netsuite:modified-listener ...`
-| On Deleted Object | `<netsuite:deleted-trigger ...` | `<netsuite:deleted-listener ...`
+| On New Object | `<netsuite:new-object-trigger` | `<netsuite:new-object-listener`
+| On Modified Object |  `<netsuite:modified-trigger` | `<netsuite:modified-listener`
+| On Deleted Object | `<netsuite:deleted-trigger` | `<netsuite:deleted-listener`
 |===
 
 == Upgrade Steps
 
-If you are upgrading from an earlier version of the connector, a small popup appears in the bottom right corner of Anypoint Studio with the `Updates Available` message.
+If you are upgrading from an earlier version of the connector:
 
-Click the pop-up and follow the prompts to install the latest version.
+. A small pop-up message appears in the bottom right corner of Anypoint Studio with the `Updates Available` message.
+. Click the pop-up message and follow the prompts to install the latest version.
 
 You can also follow these steps to perform the upgrade:
 
 . In Anypoint Studio, create a Mule project.
 . In *Mule Palette*, click *Search in Exchange*.
-. In *Add Dependencies to Project*, enter `NetSuite` in the search field.
-. In *Available modules*, select *NetSuite* and click *Add*.
-. Click *Finish*.
-. Verify that the NetSuite Connector dependency version is 10.0.0 in the `pom.xml` file in the Mule project.
-
+. In *Add Dependencies to Project*, enter `netsuite` in the search field.
+. In *Available modules*, select *NetSuite* and click *Add*. +
 Studio upgrades the connector automatically.
+. Click *Finish*.
+. In the `pom.xml` file for the Mule project, verify that the NetSuite Connector dependency version is `10.0.0`. 
 
-== Additional Notes
+== Additional Tasks
```

Comment:
Hmm...aren't each of the following already documented above?
Did you intend a "summary" or shortcut list of actions to ugpgrading here?
Please reevaluate this text in light of that above.

Copyedits only: